assets/js/components/Stepper/__snapshots__/index.test.js.snap

```diff
@@ -291,13 +255,7 @@ exports[`Stepper should render the active step as active and display its content
         </h2>
         <div
           class="googlesitekit-stepper__step-content-container"
-        >
-          <div
-            class="googlesitekit-stepper__step-content"
-          >
-            This is step 2.
-          </div>
-        </div>
+        />
```

Comment:
Looks like this test is arguably failing now, since it's not outputting anything in the content.

Reply:
Hi @tofumatt. I believe this is expected because we're only rendering the children if the current step is the active step. Also, the test doesn't seem to fail either in CI or in my local environment. Do you see the failure on your end?

Reply:
Oops, I thought no content was being output, but I see now step 1 was. It didn't appear in the diff so I missed it, not expecting both steps to have originally been output. All good 👍🏻